### PR TITLE
feat(deps): use gen3config instead of code that was in fence

### PR DESCRIPTION
**Pull request by Avantol13**
Description about what this pull request does.

Please make sure to follow the [DEV guidelines](https://gen3.org/resources/developer/dev-introduction/) before asking for review.

### Improvements
- use new gen3config instead of having all config code in fence

### CODE REVIEW

File: fence/config.py [hunk 1]
```diff
@@ -1,294 +1,21 @@
...
-            # new namespace, remove current vars (no longer available as replacements)
-            for old_cfg, value in cfg.iteritems():
-                replacements.pop(old_cfg, None)
-
-        return fully_rendered_cfgs
-    except AttributeError:
-        # it's not a dict, so lets try to render it. But only if it's
-        # truthy (which means there's actually something to replace)
-        if cfg:
-            try:
```
Comment (zflamig) [inline fence/config.py]: Can we cooy this bug fix over to gen3config? Added here https://github.com/uc-cdis/fence/commit/1806157a0dd2fdb733a8a1bb88aecf29aefe74b1#diff-adaddee2da1a1e85441bb88ec68d294b

Comment (Avantol13) [reply fence/config.py]: for sure, I'll update the library

**Comment by PlanXCyborg** [comment]
The style in this PR agrees with [`black`](https://github.com/ambv/black). :heavy_check_mark:

This formatting comment was generated automatically by a script in [uc-cdis/wool](https://github.com/uc-cdis/wool).

**Comment by Avantol13** [comment]
> Works for me! Why the Jenkins test is still failing...

I think there's an issue with the config loading for fence-create calls. I will debug and update as necessary

**Pull request merged**